src/misc.f90

```diff
@@ -1653,4 +1653,49 @@ subroutine subtitle(text)
     string2print(75 - length + 1 : 75) = text
     if(this_image() == 1) write(*,'(A75)') string2print
   end subroutine subtitle
+
+  subroutine Hilbert_transform(fx, Hfx)
+     !! Does Hilbert tranform for a given function
+     !! Ref - EQ (4)3, R. Balito et. al.
+     !! - An algorithm for fast Hilbert transform of real
+     !!   functions
+     !!
+     !! fx is the function
+     !! Hfx is the Hilbert transform of the function
+     !!
+
+     real(r64), allocatable, intent(out) :: Hfx(:)
+     real(r64), intent(in) :: fx(:)
```

Comment:
I prefer to have the `intent(in)` arguments declared before the `intent(out)` arguments.